### ISSUE TITLE
No data back from aade - probably my mistake

### ISSUE THREAD

**Issue by ratmole** [firebed/aade-mydata#10]
Hi, trying to get theRequestTransmittedDocs using this code example:
```
<?php
require_once __DIR__ . '/vendor/autoload.php';

use Firebed\AadeMyData\Http\MyDataRequest;
use Firebed\AadeMyData\Http\RequestTransmittedDocs;


ini_set('display_errors', 1);
ini_set('display_startup_errors', 1);
error_reporting(E_ALL);


$env = "prod"; // For production use "prod"
$user_id = "my-id";
$subscription_key = "my-key";

MyDataRequest::setEnvironment($env);
MyDataRequest::verifyClient(false);
MyDataRequest::setCredentials($user_id, $subscription_key);

$mark = "";
$nextPartitionKey = null;
$nextRowKey = null;

$request = new RequestTransmittedDocs();
$response = $request->handle($mark, $nextPartitionKey, $nextRowKey);

    if ($response) {
            echo "OK!";
            var_dump($response);
    } 

?>
```
and nothing returns back, is i something that im missing?
`$response => object(Firebed\AadeMyData\Models\RequestedDoc)#4 (1) { ["attributes":protected]=> array(0) { } } `

**Comment by tpharaoh** [comment]
you seem to have 2 different variables here, or is it a typo in the issue?

response

responce

Looks like your code is from docs, as is mine, and works fine!

**Comment by ratmole** [comment]
it was a typo in the issue,
are you able to fetch the invoices from aade with my code and your credentials?

**Comment by firebed** [comment]
The code is functioning correctly if you receive a response, even if it's an empty array, from aade. However, it is crucial to verify that the subscription key you utilize to retrieve transmitted invoices is the same one you used to create them. If you generated invoices with a different platform, such as the official timologio, you cannot retrieve those invoices using your REST API subscription key.

**Comment by tpharaoh** [comment]
Yes. I just copy paste from docs and works like magic…

**Comment by sksksksk** [comment]
> The code is functioning correctly if you receive a response, even if it's an empty array, from aade. However, it is crucial to verify that the subscription key you utilize to retrieve transmitted invoices is the same one you used to create them. If you generated invoices with a different platform, such as the official timologio, you cannot retrieve those invoices using your REST API subscription key.

sorry for hijacking this thread, but this is highly unexpected: so you are saying that there is no way to retrieve data with the REST API that was not created through the REST API? And only data that was created with the same credentials? I have seen applications in accountant offices (logistes) that retrieve invoice data created through the various invoice issue sites. So do these offices retrieve data? Or are we talking only about the dev platform?

**Comment by tpharaoh** [comment]
I can't speak for the dev, but only from my experience.

My accountant uses his own platform, using the same API key. And I have
used the online myData by hand. The query method pulls both.

This is quite an impressive tool. Big thanks to the dev! I built a small
Laravel tool over the weekend for my own use, since my accountant had
issues with me using stripe, and all the commercial tools were only Greek,
and my Greek is still very beginner :)

**Comment by firebed** [comment]
I apologize for my previous answer, as it contained some inaccuracies and I'll try to revise the content to clarify the situation.

There are several ways to issue invoices, including myDATA invoicing, timologio, data providers, and tax registers. Initially, I used the timologio application to issue my first invoices. However, when I switched to my own platform with a new REST API key, my accountant had to remove the timologio subscription key from his software and use the REST API key instead. This was necessary because my accountant could not retrieve my new invoices using the timologio subscription key.

Using your own REST API key allows you to retrieve invoices that were issued with timologio and probably the rest of the platforms, but not the other way around. For example, you cannot retrieve invoices generated by a data provider using the timologio subscription key. However even if you use the REST API key, there are some limitations. For example, you cannot cancel invoices that were generated using a different platform. According to the official documentation (page 48), invoices with a certain MARK cannot be cancelled if they were posted by the data provider, myDATA Invoicing, or the TaxRegister.

I hope this helps to clarify the situation.

https://www.aade.gr/sites/default/files/2022-09/myDATA%20API%20Documentation%20v1.0.6_official_erp.pdf
